recipe-server/client/control_new/utils/api.js

```diff
@@ -0,0 +1,75 @@
+import * as Cookie from 'js-cookie';
+
+
+export default class APIClient {
```

Comment:
Can you touch on why you decided to turn this into a class? I debated internally for a few minutes and I think this is a smart move, but I'd be interested to hear your thoughts on it.